addons/photo_collage/templates.xml

```diff
@@ -0,0 +1,15 @@
+<openerp>
+    <data>
+        <template id="listing">
+           <t t-call="website.layout">
+              <ul>
+                <li t-foreach="objects" t-as="photo_collage">
+                    <div class="row">
+                        <img t-attf-src="/website/image/photo_collage/{{photo_collage.id}}/attachment"/>
```

Comment:
Where did you get the `/attachment` part from? I don't see this as an option for the `/website/image/` controller...